scripts/Player.gd

```diff
@@ -39,4 +43,4 @@ func _physics_process(delta):
 	
 	# move the camera with the player
 	$CameraJoint.translation = translation
```

Comment:
where is `translation` defined?

Reply:
`translation` is a property that every object has, and it's defined by where the object is in the 3D viewport.